play/src/App.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
If we're setting up a new app, might worth exploring using Typescript instead of jsx

Reply:
Great points and thanks for going in details! I totally agree with this slowing us down and can be harder for others to ramp and start contributing to it. However, this also can be seen as an investment, especially that we're considering PWA version of the Play which means this product would become both our website and mobile version of this product. Especially with the benefits of nullability checks and being strongly types, this can help reducing bugs over time.

We should look at this as an investment to help in future to reduce bugs, take advantage of TS features to have cleaner code, and something that attracts more talent for this product as it offers modern technologies to learn and work on.

With all that said, since you're the one working on it, you should choose the technology that you feel comfortable with.

Reply:
That's a great way to frame the issue. I'm going to run this by Sinclair tomorrow and if he signs off begin transferring to TS. Thanks for the insight on this - super valuable.

Reply:
Great discussion -- thanks to you both for sharing these details. Spoke with Drew earlier, and given all the pros summarized above, TypeScript sounds like the way go -- full speed ahead!